chpl/chpl-service/src/main/java/gov/healthit/chpl/auth/authentication/JWTUserConverterFacade.java

```diff
@@ -51,6 +51,10 @@ public JWTAuthenticatedUser getAuthenticatedUser(String jwt) {
 
     @Override
     public JWTAuthenticatedUser getImpersonatingUser(String jwt) throws JWTValidationException {
-        throw new NotImplementedException("CognitoJwtUserConverter.getImpersonatingUser() has not been implemented.");
+        if (ff4j.check(FeatureList.SSO)) {
```

Comment:
Does this mean we cannot impersonate, at all, if the SSO flag is on? I know we're not turning it on in PROD, but I thought we were leaving it on in DEV/QA/STG for burn-in / testing purposes. If that's the case, we've basically turned off impersonation